services/ows_refactored/land_cover/ows_esa_worldcover_cfg_2021.py

```diff
@@ -0,0 +1,125 @@
+    "needed_bands": ["classification"],
+    "index_function": {
+        "function": "datacube_ows.band_utils.single_band",
+        "mapped_bands": True,
+        "kwargs": {
+            "band": "classification",
+        },
+    },
+    "value_map": {
+        "classification": [
+            {
+                "title": "",
+                "abstract": "",
+                "values": [0],
+                "alpha": 0.0,
+                "color": "#707070",
+            },
+            {
+                "title": "Tree cover",
+                "abstract": "",
+                "values": [10],
+                "color": "#006400",
+            },
+            {
+                "title": "Shrubland",
+                "abstract": "",
+                "values": [20],
+                "color": "#ffbd22",
+            },
+            {
+                "title": "Grassland",
+                "abstract": "",
+                "values": [30],
+                "color": "#ffff4c",
+            },
+            {
+                "title": "Cropland",
+                "abstract": "",
+                "values": [40],
+                "color": "#ef96ff",
+            },
+            {
+                "title": "Built-up",
+                "abstract": "",
+                "values": [50],
+                "color": "#fa0000",
+            },
+            {
+                "title": "Bare/sparse vegetation",
+                "abstract": "",
+                "values": [60],
+                "color": "#b4b4b4",
+            },
+            {
+                "title": "Snow and ice",
+                "abstract": "",
+                "values": [70],
+                "color": "#f0f0f0",
+            },
+            {
+                "title": "Permanent water bodies",
+                "abstract": "",
+                "values": [80],
+                "color": "#0064c8",
+            },
+            {
+                "title": "Herbaceous wetland",
+                "abstract": "",
+                "values": [90],
+                "color": "#0095a0",
+            },
+            {
+                "title": "Mangroves",
+                "abstract": "",
+                "values": [95],
+                "color": "#00cf75",
+            },
+            {
+                "title": "Moss and lichen",
+                "abstract": "",
+                "values": [100],
+                "color": "#fae6a0",
+            },
+        ]
+    },
+    "legend": {"width": 3.0, "height": 4.0},
+}
+
+layer = {
+    "title": "ESA WorldCover 2021",
+    "name": "esa_worldcover_2021",
+    "abstract": """
+The ESA WorldCover 2021 product is a global, 10 m resolution land cover product, which Digital Earth Africa is making accessible over Africa.
+
+The product uses Sentinel-1 and Sentinel-2 data to produce 11 land cover classes, according to the GAO LCCS schema.
+
+For more information, see https://esa-worldcover.s3.amazonaws.com/v100/2020/docs/WorldCover_PUM_V1.0.pdf
```

Comment:
is there a different document for the 2021 product?

Reply:
I found it - https://worldcover2021.esa.int/data/docs/WorldCover_PUM_V2.0.pdf. I will update it.